HenHen.Core/Worlds/Plants/Flower.cs

```diff
@@ -14,16 +14,16 @@ public class Flower : Plant
         public Flower(FlowerBreed breed) : base(breed)
         {
             var random = new Random();
-            var randomPoint = random.NextDouble();
-            var currentEndPoint = 0.0;
+            var currentEndPoint = 0;
             var suma = 0.0;
             for (var i = 0; i < breed.DropAmount.Count; i++)
             {
                 suma += breed.DropAmount[i].chance;
             }
```

Comment:
A shorter notation would be
```cs
var sum = breed.DropAmount.Sum(ac => ac.amount);
```
The LINQ Sum has to be imported (<kbd>Alt</kbd><kbd>Enter</kbd>).